operatorcourier/verify.py

```diff
@@ -0,0 +1,143 @@
+import os
+import logging
+from typing import List, Dict
+import semver
+from operatorcourier.build import BuildCmd
+from operatorcourier.validate import ValidateCmd
+from operatorcourier.errors import OpCourierBadBundle
+from operatorcourier.format import format_bundle
+from operatorcourier import identify
+
+logger = logging.getLogger(__name__)
+FLAT_KEY = '__flat__'
+
+
+class Verification:
```

Comment:
Verification seems like the wrong name -- we already have a thing called `Validate` which seems very similar (although they serve very different meanings). Also, this thing does not actually _do_ any of the verification. Maybe this is a `Manifest` or `Verified_Manifest` [(from operator-registry)](https://github.com/operator-framework/operator-registry#manifest-format)